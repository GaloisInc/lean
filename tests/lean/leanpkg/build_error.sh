cd build_error
leanpkg build
