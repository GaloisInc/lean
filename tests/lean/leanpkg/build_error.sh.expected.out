configuring build_error 0.1
> lean --make src
err.lean:1:0: error: command expected
external command exited with status 1
1
